### PR TITLE
Enable RDS event notification

### PR DESCRIPTION
**Pull request by lzrocha**
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to <repo_name>?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the CONTRIBUTING.md doc.
- [x] I have added necessary documentation (if appropriate).
- [x] Any dependent changes have been merged and published in downstream modules.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: event_subscription.tf [hunk 1]
```diff
@@ -0,0 +1,11 @@
+resource "aws_db_event_subscription" "rds_events" {
+  count     =  length(var.identifier) > 0 ? 1 : 0
+  name      = "${var.account_name}-${var.identifier}-rds-event-sub"
+  sns_topic = var.alarm_sns_topics
+
+  source_type = "db-instance"
+  source_ids  = [var.identifier]
+  
+  event_categories = var.event_categories == null ? split(",",
+    "availability,deletion,failover,failure,low storage,maintenance,notification,read replica,recovery,restoration") : split(",",var.event_categories)
```
Comment (jeremiasroma) [inline event_subscription.tf:10]: Can this list be added as a default value in the variable instead of having the value as null?

**Review by jeremiasroma — CHANGES_REQUESTED** [review]

File: event_subscription.tf (hunk 1 above)
Comment (lzrocha) [reply event_subscription.tf]: well spotted, let me update

File: cloudwatch-alarms.tf [hunk 2]
```diff
@@ -8,8 +8,8 @@ resource "aws_cloudwatch_metric_alarm" "low_memory" {
   statistic           = "Maximum"
   threshold           = "100"
   alarm_description   = "Database instance memory above threshold"
-  alarm_actions       = var.alarm_sns_topics
-  ok_actions          = var.alarm_sns_topics
+  alarm_actions       = [var.alarm_sns_topics]
```
Comment (Renatovnctavares) [inline cloudwatch-alarms.tf:11]: I think `alarm_sns_topics` is a list already,  pls look at the variables file 
module is expecting `[sns1,sns2]`   if you are changing it for string type you should update the variables file as well.

**Review by jeremiasroma — COMMENTED** [review]
LGMT

**Review by jeremiasroma — DISMISSED** [review]

File: cloudwatch-alarms.tf (hunk 2 above)
Comment (lzrocha) [reply cloudwatch-alarms.tf:11]: done

File: event_subscription.tf [hunk 3]
```diff
@@ -0,0 +1,11 @@
+resource "aws_db_event_subscription" "rds_events" {
+  count     =  length(var.identifier) > 0 ? 1 : 0
```
[the remark this answers is not in the document]
Comment (lzrocha) [reply event_subscription.tf]: No, you can't. You can disable the event subscription only.

**Review by adenot — APPROVED** [review]

**Pull request merged**